controllers/grid/files/review/AuthorReviewRevisionsGridHandler.inc.php

```diff
@@ -29,11 +29,97 @@ function __construct() {
 
 		$this->addRoleAssignment(
 			array(ROLE_ID_AUTHOR),
-			array('fetchGrid', 'fetchRow')
+			array('fetchGrid', 'fetchRow','sendRevisions')
 		);
 
 		$this->setTitle('editor.submission.revisions');
 	}
+        
+        function initialize($request) {
```

Comment:
Make sure you include a header on all functions -- even if it's something brief like...

    /**
     * @copydoc GridHandler::initialize()
     */